sycl/test/abi/symbol_size.cpp

```diff
@@ -69,13 +68,6 @@ int main() {
 #ifdef __SYCL_DEVICE_ONLY__
   check_size<private_memory<int, 1>, 4>();
   check_size<detail::sampler_impl, 8>();
```

Comment:
Should we bump library version with this change?

Reply:
It's tricky question. But, I think, it's safer to bump. Will do.

Reply:
Done.